src/Components/People.js

```diff
@@ -0,0 +1,134 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import "./People.css";
+
+const People = () => {
+  const [userInput, setUserInput] = useState("");
+  const [name, setName] = useState("");
+  const [age, setAge] = useState("");
+  const [gender, setGender] = useState("");
+  // const [people, setPeople] = useState("");
+
+  const fetchPeople = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/people");
+      // debugger;
+      res.data.forEach((person) => {
```

Comment:
For this, you're better off using a filter or a find method instead of forEach. Will make accessing the information much easier